roda-core/roda-core/src/main/java/org/roda/core/common/PremisV3Utils.java

```diff
@@ -273,6 +280,54 @@ private static ExtensionComplexType getTechnicalMetadata(gov.loc.premis.v3.File
     return extensionComplexType;
   }
 
+  public static boolean formatWasManuallyModified(ModelService model, String aipId, String representationId,
+    List<String> fileDirectoryPath, String fileId, String username) {
+    Binary premisBin = null;
+
+    try {
+      try {
+        premisBin = model.retrievePreservationFile(aipId, representationId, fileDirectoryPath, fileId);
+      } catch (NotFoundException e) {
+        LOGGER.debug("PREMIS object skeleton does not exist yet. Creating PREMIS object!");
+        List<String> algorithms = RodaCoreFactory.getFixityAlgorithms();
+
+        if (fileId == null) {
+          PremisSkeletonPluginUtils.createPremisSkeletonOnRepresentation(model, aipId, representationId, algorithms,
+            username);
+        } else {
+          // File file;
+          // if (shallow) {
+          // file = model.retrieveFileInsideManifest(aipId, representationId,
+          // fileDirectoryPath, fileId);
+          // } else {
+          // file = model.retrieveFile(aipId, representationId, fileDirectoryPath,
+          // fileId);
+          // }
```

Comment:
If the code is not used please remove

Reply:
Resolved in https://github.com/keeps/roda/pull/3255/commits/de2e94ce3f2ffe9099904d69b8e10203caf3a2fb